app/inputs/multi_value_input.rb

```diff
@@ -0,0 +1,64 @@
+class MultiValueInput < SimpleForm::Inputs::Base
```

Comment:
Looks like it - https://github.com/projecthydra/hydra-editor/blob/master/spec/inputs/multi_value_input_spec.rb

I could pull that in or we could get this pushed up and test it there.